Server/SQL/schema.sql

```diff
@@ -15,7 +15,7 @@ CREATE TABLE IF NOT EXISTS Deck
 (
     uuid VARCHAR(36) NOT NULL PRIMARY KEY,
     created TIMESTAMP NOT NULL DEFAULT NOW(),
-    last_update TIMESTAMP NOT NULL,
+    last_update TIMESTAMP NULL,
```

Comment:
Did this line work for you when you built? Its was set as not null, but given no default, so it wouldn't build for me. And since we can't set it to Now() because of limitations, I figured we might as well let it be null